testsuite/integration-arquillian/servers/auth-server/quarkus/pom.xml

```diff
@@ -186,6 +186,31 @@
                             </artifactItems>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>includeProprietaryDependencies</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>com.oracle.database.jdbc</groupId>
+                                    <artifactId>ojdbc11</artifactId>
+                                    <version>${oracle-jdbc.version}</version>
+                                    <type>jar</type>
+                                    <outputDirectory>${auth.server.home}/providers</outputDirectory>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>com.oracle.database.nls</groupId>
+                                    <artifactId>orai18n</artifactId>
+                                    <version>${oracle-jdbc.version}</version>
+                                    <type>jar</type>
+                                    <outputDirectory>${auth.server.home}/providers</outputDirectory>
+                                </artifactItem>
+                            </artifactItems>
+                        </configuration>
+                    </execution>
```

Comment:
We will add the Oracle JDBC driver to the `providers` folder of the dist even when the Oracle JDBC driver is present via transitive dependency of the Quarkus Oracle JDBC extension. We will avoid further differentiation between `main` and `22.x`. As the driver will be only part of 22.x, I don't see any benefit to making other workarounds bcs of testsuite.

@vmuzikar Is it ok for you?

Reply:
If we add this to the `product` profile, would it have some disadvantages?

Reply:
@mhajas ahh, probably not. Better approach.